manifests/charts/istio-cni/templates/clusterrole.yaml

```diff
@@ -73,4 +73,8 @@ rules:
   {{- /* pods/status is less privileged than the full pod, and either can label. So use the lower pods/status */}}
   resources: ["pods/status"]
   verbs: ["patch", "update"]
+- apiGroups: ["apps"]
+  resources: ["daemonsets"]
+  resourceNames: ["{{ template "name" . }}-node"]
+  verbs: ["get", "list"]
```

Comment:
I don't think it matters much but curious -- do we need list?

Reply:
I'll try without, mostly added out of habit since kube clients seem to frequently need both.